pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -156,8 +156,20 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 			},
 			Rules: []authorizationapi.PolicyRule{
```

Comment:
In the rolling updater?

----- Original Message -----
> > @@ -156,8 +156,20 @@ func GetBootstrapClusterRoles()
> > []authorizationapi.ClusterRole {
> >  			},
> >  			Rules: []authorizationapi.PolicyRule{
> 
> This just in: throw in `list` for pods as well, since I'll be switching to a
> reflector/FIFO setup for the pod watch functionality.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/2477/files#r31065030